src/benchmark_runner.cc

```diff
@@ -227,10 +228,8 @@ class BenchmarkRunner {
     // And get rid of the manager.
     manager.reset();
 
-    // Adjust real/manual time stats since they were reported per thread.
-    i.results.real_time_used /= b.threads;
-    i.results.manual_time_used /= b.threads;
-    // If we were measuring whole-process CPU usage, adjust the CPU time too.
+    // If we were measuring whole-process CPU usage then each thread reports
+    // total CPU time of all threads. Divide by threads to get real value.
     if (b.measure_process_cpu_time) i.results.cpu_time_used /= b.threads;
```

Comment:
Shouldn't this also go away, don't we now divide it twice?

Reply:
Nope. Each thread reports total process time individually and they are summed i.e. multiplied by thread count.